README.md

```diff
@@ -14,3 +14,9 @@ Development Installation
 1. Make sure you have the development dependencies installed
 2. Place `septa_bus.zip`, `septa_rail.zip`, and `patco.zip` in the gtfs folder
 3. Run `vagrant up`
+4. `vagrant ssh`
+5. `. /app_env/activate`
```

Comment:
Yeah, we don't normally put virtualenvs inside of Vagrant boxes since they serve overlapping purposes. The ones you've probably seen (Cicero and OpenDataPhilly) have them for historical reasons. Cicero is old and was migrated onto Vagrant, while with ODP we used pre-existing installation scripts / packages to save time and there was no reason to mess with it once it was working. It's definitely not a big deal though.